contracts/Collateralizer.sol

```diff
@@ -347,7 +348,7 @@ contract Collateralized is TermsContract {
         // Since this value takes the rightmost place in the parameters string,
         // we do not need to bit-shift it.
         bytes32 gracePeriodInDays =
-            parameters & 0x00000000000000000000000000000000000000000000000000000000000000ff;
+        parameters & 0x00000000000000000000000000000000000000000000000000000000000000ff;
```

Comment:
Nit: inconsistent styling with the rest of the codebase on all three of the `parameters & _` statements